documentation/src/main/asciidoc/topics/proc_creating_cache_rest.adoc

```diff
@@ -0,0 +1,36 @@
+= Creating Caches
+{brandname} servers do not provide default caches. To store data in your cluster, you must first define a cache.
+
+.Prerequisites
+A REST client. The following procedures use `curl` but you can use any client.
```

Comment:
As soon as we merge the CLI, this paragraph can be replaced with instructions using that